h/templates/includes/annotation_card.html.jinja2

```diff
@@ -29,5 +29,10 @@
   <section class="annotation-card__text">
     {{ annotation.text_rendered|safe }}
   </section>
+  <footer class="annotation-card__footer">
+    <a class="annotation-card__incontext-link" href="{{ incontext_link }}" target="_blank" title="{{ incontext_link }}">
```

Comment:
`incontext_link` is sometimes `None` (the `links.incontext_link()` function returns `None` for some annotations that can't be linked to), I think you might need an `{% if %}` here to not render the link in that case?